.github/workflows/release.yml

```diff
@@ -0,0 +1,58 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*'
+  pull_request:
+
+
+jobs:
+  # taken from https://github.com/docker/build-push-action/blob/master/UPGRADE.md#tags-with-ref-and-git-labels
+  docker:
+    name: Publish Docker image
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'returntocorp'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get custom tags
+        id: custom_tags
+        run: |
+          CUSTOM_TAGS=""
+          if [[ $GITHUB_REF == refs/tags/v1 ]]; then
+            CUSTOM_TAGS="stable"
+          fi
+          echo ::set-output name=custom_tags::${CUSTOM_TAGS}
+      - name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1.11.0
+        with:
+          images: returntocorp/semgrep-action,returntocorp/semgrep-agent
```

Comment:
Are we pushing to two images?

Reply:
We had semgrep-action since the start and started publishing to -agent when expanding beyond GitHub Actions.